.spelling

```diff
@@ -144,3 +144,11 @@ v3
 publicised
 analyse
 PKCS#12
+OperatorHub
+keystores
+keystores
+keystore
+config
+cmctl
+RedHat
```

Comment:
Can we just refer to `Red Hat` instead?

Reply:
I always was so convinced it was RedHat.... turns out you should never trust yourself